website/docs/Migration20.md

```diff
@@ -39,9 +39,29 @@ Both changes should improve code readibility.
 Please note that in many cases `waitFor` call can be replaced by proper use of `findBy` asynchonous queries resulting in more streamlined test code.
 :::
 
-## Removed global `debug` function
+## Removed global debug functions
 
-Global debug function has been removed in favor of `debug()` method returned from `render()` function.
+Global `debug()` and `shallow()` functions have been removed in favor of `debug()` method returned from `render()` function.
```

Comment:
Let's add a separate section for shallow:
```
## Removed `shallow` helper

Shallow rendering React component is usually not a good idea, so we decided to remove the API. However, if you find it useful or need to support legacy tests, feel free to implement it yourself. Here's a sample implementation:

```js
import ShallowRenderer from 'react-test-renderer/shallow';

export function shallow(
  instance: ReactTestInstance | React.Element<any>
) {
  const renderer = new ShallowRenderer();
  renderer.render(React.createElement(instance.type, instance.props));

  return { output: renderer.getRenderOutput() };
}
```
```

Reply:
Done